CONTRIBUTORS.txt

```diff
@@ -222,3 +222,5 @@ under this name, or we did not manage to find their commits in the history.
 - carl
 - alain lefroy
 - Mark Gius
+- Robert Hofer <1058012+hofrob@users.noreply.github.com>
+- Emmanuel Ferdman <emmanuelferdman@gmail.com>
```

Comment:
I'm always doing ``rm CONTRIBUTORS.txt;python3 script/create_contributor_list.py`` then doing a git add . -p which i'm going to do now :D

Reply:
I have no idea what's going on here honestly, so thanks for doing it.

Reply:
The contributors are ordered based on number of commit (you have more commit in astroid than Daniel now, congrat :) )

Reply:
Congrats! 😁